File: Day_29/Q113.java
import java.util.*;
class Q113{
    static Scanner sc = new Scanner(System.in);

    //Addition
    static void addition(){
        double a,b;
        System.out.print("Enter two numbers: ");
        a = sc.nextDouble();
        b = sc.nextDouble();
        System.out.println("Sum = " + (a + b));
    }

    //Subtraction
    static void subtraction(){
        double a,b;
        System.out.print("Enter two numbers: ");
        a = sc.nextDouble();
        b = sc.nextDouble();
        System.out.println("Difference = " + (a - b));
    }

    //Multiplication
    static void multiplication(){
        double a,b;
        System.out.print("Enter two numbers: ");
        a = sc.nextDouble();
        b = sc.nextDouble();
        System.out.println("Product = " + (a * b));
    }

    //Division
    static void division(){
        double a,b;
        System.out.print("Enter two numbers: ");
        a = sc.nextDouble();
        b = sc.nextDouble();
        if(b == 0) System.out.println("Division by zero is not possible");
        else System.out.println("Quotient = " + (a / b));
    }

    //Menu
    static void menu(){
        System.out.println("\n1. Addition");
        System.out.println("2. Subtraction");
        System.out.println("3. Multiplication");
        System.out.println("4. Division");
        System.out.println("5. Exit");
        System.out.print("Enter your choice: ");
    }

    public static void main(String[] args){
        int choice;
        do{
            menu();
            choice = sc.nextInt();
            switch(choice){
                case 1: addition(); break;
                case 2: subtraction(); break;
                case 3: multiplication(); break;
                case 4: division(); break;
                case 5: System.out.println("Program ended"); break;
                default: System.out.println("Invalid choice");
            }
        }while(choice != 5);
        sc.close();
    }
}

File: Day_29/Q113.cpp
#include <bits/stdc++.h>
using namespace std;

//Addition
void addition(){
    double a,b;
    cout << "Enter two numbers: ";
    cin >> a >> b;
    cout << "Sum = " << a + b << endl;
}

//Subtraction
void subtraction(){
    double a,b;
    cout << "Enter two numbers: ";
    cin >> a >> b;
    cout << "Difference = " << a - b << endl;
}

//Multiplication
void multiplication(){
    double a,b;
    cout << "Enter two numbers: ";
    cin >> a >> b;
    cout << "Product = " << a * b << endl;
}

//Division
void division(){
    double a,b;
    cout << "Enter two numbers: ";
    cin >> a >> b;
    if(b == 0) cout << "Division by zero is not possible" << endl;
    else cout << "Quotient = " << a / b << endl;
}

//Menu
void menu(){
    cout << "\n1. Addition" << endl;
    cout << "2. Subtraction" << endl;
    cout << "3. Multiplication" << endl;
    cout << "4. Division" << endl;
    cout << "5. Exit" << endl;
    cout << "Enter your choice: ";
}

int main(){
    int choice;
    do{
        menu();
        cin >> choice;
        switch(choice){
            case 1: addition(); break;
            case 2: subtraction(); break;
            case 3: multiplication(); break;
            case 4: division(); break;
            case 5: cout << "Program ended" << endl; break;
            default: cout << "Invalid choice" << endl;
        }
    }while(choice != 5);
    return 0;
}
